config/default.yaml

```diff
@@ -56,6 +56,7 @@ httpd:
 
 datastore:
   plugin: sequelize
+  ddlSyncEnabled: "true"
```

Comment:
Why are we using `string`?

Reply:
followed buildMetricsEnabled. I think boolean will not work in k8s env variables